Hummingbird.docc/Articles/ServerProtocol.md

```diff
@@ -0,0 +1,122 @@
+# Server protocol
```

Comment:
"Server protocol" as an article title is pretty arbitrary - it could mean anything if you just look at the title. Would "Enabling HTTP/2, TLS and WebSockets" be a better title?